syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTCurrencyParams";
option optimize_for = LITE_RUNTIME;

message CurrencyParams
{
    optional uint32 version = 1;
    optional string TLA = 2;
    optional uint32 power = 3;
    optional string fraction = 4;
}
